crates/biome_js_analyze/src/lint/nursery/no_label_without_control.rs

```diff
@@ -0,0 +1,290 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_deserialize_macros::Deserializable;
+use biome_js_syntax::{
+    AnyJsxAttribute, AnyJsxAttributeName, AnyJsxAttributeValue, AnyJsxTag, JsxAttribute,
+    JsxAttributeList, JsxName, JsxReferenceIdentifier, JsxText,
+};
+use biome_rowan::AstNode;
+use serde::{Deserialize, Serialize};
+
+declare_rule! {
+    /// Enforce that a label element or component has a text label and an associated control.
```

Comment:
Maybe "input" instead of "control"? While control is generic, it is **too** generic and maybe uses don't know what it is (myself)

Reply:
I changed all the references to input and added an explanation stating that an "input" is actually more than just the element: 
```
An "input" is considered one of the following elements: `input`, `meter`, `output`, `progress`, `select` or `textarea`.
```